package/libyui-rest-api.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed Feb  5 12:09:35 UTC 2020 - Oleksandr Orlov <oorlov@suse.com>
+
+- Replace hard-coded HTML documentation with the url to the actual
+  documentation in project repo
+- Add "/version" endpoint to access to API version
```

Comment:
Please add a note that the API is versioned now and uses the `/v1/` prefix in the URL path.